src/Compiler/CompilerUtils.cpp

```diff
@@ -521,6 +521,18 @@ static void genLLVMBitcode(const mlir::OwningOpRef<ModuleOp> &module,
     llvm::errs() << "Failed to translate module to LLVMIR.\n";
     exit(1);
   }
+
+  // Emit metadata "zos_le_char_mode" for z/OS. Use EBCDIC codepage by default.
+  bool zOS = StringRef(getTargetTripleOption()).endswith_insensitive("zos");
+  if (zOS) {
+    StringRef charModeKey = "zos_le_char_mode";
+    StringRef charModeValue = "ebcdic";
```

Comment:
[nit]: you could sink this line inside the if stmt

Reply:
Done.

Reply:
hmmm I still see it in at line 528.  Forgot to push ?

Reply:
Done. I though it was the above if.